README.md

```diff
@@ -156,7 +161,7 @@ By default, [`odin`](#api) containers can only provide instances of `class`, so
 
 This snippet shows how to create/obtain a container:
 ```typescript
-import { odin } from '@odinjs/odin';
+import { odin } from '@philips-software/odin';
 
 class Injectable { }
 
```

Comment:
I think `bundle.register(Injectable)` can also be deprecated. Therefore, we have two option for the snippet below.

1. Keep it as it is, it just points out the deprecation through a note.
2. Change it to already register using the `options` param along with the `class` name.